dFusion/dFusionSpec.md

```diff
@@ -42,28 +45,45 @@ Accounts will pay "rent" to occupy an active account. The account index can be u
 Furthermore, we store a bi-map of token address to its index `0 <= t <= T`, for each token that can be traded on the exchange.
 When specifying tokens inside orders, deposits and withdrawel requests, we use the token's index `0 <= t <= T` instead of the full address.
 
-As orders, deposits and withdrawl requests are collected they are not directly stored in the smart contract.
-Doing so would require a `SSTORE` instruction which would be too gas-expensive.
+As limit orders, deposits and withdrawl requests are collected they are not directly stored in the smart contract.
```

Comment:
Ben and I figured yesterday, that we need to store the withdrawl information, as we need to do the payouts based on them.